lib/jbuilder/collection_renderer.rb

```diff
@@ -60,51 +22,29 @@ def each_with_info
     end
 
     private_constant :ScopedIterator
-  end
-
-  if defined?(::ActionView::CollectionRenderer)
-    # Rails 6.1 support:
-    class CollectionRenderer < ::ActionView::CollectionRenderer # :nodoc:
-      include CollectionRenderable
 
-      def initialize(lookup_context, options, &scope)
-        super(lookup_context, options)
-        @scope = scope
-      end
-
-      private
-        def collection_with_template(view, template, layout, collection)
-          super(view, template, layout, ScopedIterator.new(collection, @scope))
-        end
+    def initialize(lookup_context, options, &scope)
```

Comment:
This initialize was part of CollectionRenderer - since you're removing its definition is this initialize still needed?

Reply:
So this is still needed?